Add Social media handbook page, fold LinkedIn beneath it (#18386)
PostHog/posthog.com: contents/handbook/content/social-media.md

---
title: Social media
sidebar: Handbook
showTitle: true
---

## Where we post

> PostHog has a presence on the following platforms: [X](https://x.com/posthog), [LinkedIn](https://www.linkedin.com/company/posthog), [YouTube](https://www.youtube.com/@posthog), [Instagram](https://www.instagram.com/teamposthog), [Threads](https://www.threads.com/@teamposthog), and [Bluesky](https://bsky.app/profile/posthog.com). Day to day, this is managed by <TeamMember name="Liam Graham" photo />.

Reviewer: ```suggestion
PostHog has a presence on [X](https://x.com/posthog), [LinkedIn](https://www.linkedin.com/company/posthog), [YouTube](https://www.youtube.com/@posthog), [Instagram](https://www.instagram.com/teamposthog), [Threads](https://www.threads.com/@teamposthog), and [Bluesky](https://bsky.app/profile/posthog.com). Day to day, this is managed by <TeamMember name="Liam Graham" photo />.
```

> Our 'primary' platforms are X, LinkedIn, and YouTube. These are those which we optimize for. Content is posted on secondary platforms, but it's not worth the resources needed to tailor content for them.

Reviewer: ```suggestion
Our primary platforms are X, LinkedIn, and YouTube. We optimize for them. Content is posted on secondary platforms, but it's not worth the resources needed to tailor content for them.
```

If the answer is "yes", we have to go back to the drawing board, either in terms of the content itself or the copy that accompanies it.

The order of effectiveness for assets is video > graphics/photos > text only, and we prioritize what's posted accordingly.

Unless otherwise stated, we optimize for a mobile viewing experience, as that's where approximately 80% of our impressions on social media come from. That means photos in 4x5, and video in 9x16.

We are not overly-concerned with everything PostHog-related needing to come from the PostHog brand accounts. On some platforms, LinkedIn especially, this is actually a hindrance: content performs much better and is perceived more earnestly when it comes from an individual.

Posts from the brand account follow our [style guide](/handbook/content/posthog-style-guide), except if it's a reply.

Replies should be done in lowercase. This humanizes the brand slightly as it underscores that a human would have written it, and is in the same style that [James](https://x.com/james406) uses on social media. More details on this can be found on the [voice & tone handbook page](/handbook/brand/tone).

## What metrics we look at

> Not a lot. We are of the belief that [all social media metrics are bad](/blog/social-media-metrics-are-all-bad). That's not to mean that they are meaningless or are not worth drawing insights from. However, social media metrics can be very easily misconstrued to draw false conclusions, and often the performance of certain posts is to do with factors beyond the platform that they're on.

Reviewer: ```suggestion
Not a lot. We believe [all social media metrics are bad](/blog/social-media-metrics-are-all-bad). They aren't meaningless or not worth drawing insights from, but can be easily misconstrued to draw false conclusions. The performance of certain posts is to do with factors beyond the platform that they're on.
```

## How to run social media for PostHog

There are two places to check a few times a day for opportunities to interact and repost: one is via each app's notifications. The other is the `#brand-mentions` Slack channel, where Octolens pulls in brand mentions from across several platforms. This channel is especially useful as it picks up mentions where we are not explicitly tagged.

On weekends at 11am PT, `#weekend-brand-mentions` has a summary of the 10 most relevant mentions we've received in the past 24 hours posted by a friendly bot.

To schedule posts in advance where we can, we use [Typefully](https://typefully.com). Ask <TeamMember name="Liam Graham" photo /> for a link to the team account.

### Posting on X

We can think of content we put out on X in two silos: one is articles, and one is everything else.

X articles have performed very well for us on the platform, unlike on LinkedIn, so any time a blog post or newsletter goes out, we are default yes on whether or not we should cross-post it to X. When sharing X articles:

> - Use aggressive header styling (the largest subheaders in a blog post should be H1 in X's article composer).

Reviewer: ```suggestion
- Use aggressive header styling (H2s in blog posts should be "Titles" in X's article composer).
```
